contrib/systemd/etcd2-backup/README.md

```diff
@@ -0,0 +1,65 @@
+# etcd2-backup
+
+Remote backup and multi-node restore services for etcd2 clusters.
+
+## Terminology
+
+**founding member** : The node which is the first member of the new recovered cluster. It is this node's rclone backup data (only) that will be used to restore the cluster. The rest of the nodes will join the cluster with no data, and simply catch up with the **founding member**.
+
+## Configuration
+Before you get started, you need to configure `30-etcd2-backup-restore.conf`
+
+```
+[Service]
+Environment="ETCD_RESTORE_MASTER_ADV_PEER_URLS=<http://host:port>"
+Environment="RCLONE_ENDPOINT=endpoint-name:path/to/backups"
+```
+
+Assuming you're deploy to CoreOS with etcd2, you should only need to change
+
+- `ETCD_RESTORE_MASTER_ADV_PEER_URLS`
+   This is the advertised peer url of the etcd2 node that will be the founding member of the new restored cluster. We will call this node the **founding member**.
```

Comment:
Is this the _new_ peer url or the _previous_ peer url (assuming you will migrate IPs during this recovery). Might be good to include an example.